### PR TITLE
Fix tab switching delay

### LINKED ISSUE #394: Fix tab switching delay
When switching between the various tabs on the signed in interface, there is a significant delay which lets the user see a different page in between switching. We need to make sure this doesn't happen.

### PR DESCRIPTION
**Pull request by katmartel**
From what I was able to test out, the problem seemed to be with routing and the auth page, since auth was constantly checking whether the logged in user had verified their email every time the button was clicked to navigate to the next page. While it was routing, the user would be null for a slight second, before it grabbed the user's information (hence why it showed the unverified email page). To by pass routing completely, I used ant design's 'Tab' component (and added icons with it to try and enhance how it looks).

### CODE REVIEW

**Review by aleksssk — APPROVED** [review]
Looks good!

**Review by wrc9817 — APPROVED** [review]
Nice

**Review by jpabadir — APPROVED** [review]
Excellent!

File: src/front-end/src/components/MentorHome.js [hunk 1]
```diff
@@ -1,46 +1,57 @@
 import React, { Component } from 'react';
-import { Button } from 'antd';
-import { Route } from 'react-router-dom';
+import { Tabs } from 'antd';
+import {
+  CarryOutOutlined,
+  SolutionOutlined,
+  UsergroupAddOutlined,
+} from '@ant-design/icons';
+// import { Route } from 'react-router-dom';
```
Comment (jpabadir) [inline src/front-end/src/components/MentorHome.js]: I would delete this.

**Pull request merged**